changelog.json

```
@@ -1,5 +1,12 @@
 {
   "releases": {
+    "2.9.13-beta1": [
+      "[New] Add support for notifications of pull request reviews - #14175",
+      "[Fixed] Pull requests adhere to temporal laws again - #14238",
+      "[Fixed] Fix repository group header overflow when text is too long - #14233. Thanks @tsvetilian-ty!",
+      "[Fixed] Clone dialog \"Choose\" button uses an open dialog for directory selection on macOS - #12812",
```

:D - my bad on writing the original note.. but is windows that is changing not mac.

```suggestion
      "[Fixed] Clone dialog \"Choose\" button uses an open dialog for directory selection on windows - #12812",
```

---

.... So I thought I left this comment yesterday when the pr was opened.. but it was still pending. :face-palm:

---

Fixed in [ee57829](https://github.com/desktop/desktop/pull/14245/commits/ee5782916f73a517a71762b9ed1909873fbfcbd5)!